### ISSUE TITLE
[Misc]: Does vllm support scenarios where multiple rounds of inferring the same model require loading the weights of that model only once?

### ISSUE THREAD

**Issue by 5jww** [vllm-project/vllm#11014]
### Anything you want to discuss about vllm.

As the title write:
Sometimes i need to run the same model multiple times, but reloading the weights for that model each time is too time consuming. So i want to know if vllm can just load weight one time?

### Before submitting a new issue...

- [X] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by DarkLight1337** [comment]
For offline inference, as long as the `LLM` instance remains in memory, you can keep using the same model weights in your code.
